pkg/kubelet/kubelet.go

```diff
@@ -1248,6 +1248,11 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 	// handled by pod workers).
 	go wait.Until(kl.podKiller, 1*time.Second, wait.NeverStop)
 
+	// Start gorouting responsible for checking limits in resolv.conf
+	if kl.resolverConfig != "" {
+		go wait.Until(func() { kl.checkLimitsForResolvConf() }, 30*time.Second, wait.NeverStop)
```

Comment:
I thought you wanted to trigger node's level event as well on pod's creation, now I see it's not the case, so removed added call from  kl.syncLoopIteration in case of ADD op. Re gorouting forking, see comment below.